test/system_test.py

```diff
@@ -15,6 +13,11 @@
 from src import machine_name, log_with_spack, sanitized_filename
 
 
+@pytest.fixture(scope="function")
+def prepost():
+    return '/scratch/mch/leclairm/uenvs/images/pre-post_v0.sqfs'
```

Comment:
Consider using a global, const variable for this.
`XXX = '/scratch/mch/leclairm/uenvs/images/pre-post_v0.sqfs'`